test/tools/runner/config.ts

```diff
@@ -250,7 +249,7 @@ export class TestConfiguration {
     options = Object.assign({}, options);
     const hosts =
       host == null ? [].concat(this.options.hostAddresses) : [new HostAddress(`${host}:${port}`)];
-    return new Topology(hosts, options);
+    return topologyWithPlaceholderClient(hosts, options);
```

Comment:
`newTopology` is used in one file: topology.test.js, can we delete this function and change that test to use `newClient`. TestConfiguration is for integration tests, so they'll almost always want a real client.